### ISSUE TITLE
Calendar won't display

### ISSUE THREAD

**Issue by finkikun** [Serhioromano/bootstrap-calendar#560]
I tried to solve it myself, but I'm stuck and don't have any idea what to do - I just got a blank page and that's all.
I present my code below - any idea, what's wrong?

![shot](https://cloud.githubusercontent.com/assets/13593288/10517733/8f793c7a-735e-11e5-8af0-0c59cb565898.jpg)

**Comment by eleazan** [comment]
Maybe... Its a wrong value for tmpl_path...

You see any error on console¿

**Comment by finkikun** [comment]
Unfortunately - there's no error in console, it's going through smoothly. That's why it's puzzling me, because theoretically everything's ok and I don't know where should I look for mistakes.

**Comment by eleazan** [comment]
So, theoretically....

You have a "domain.com", so, you have an public templates that are "domain.com/PATH/bootstrap-calendar-master/tmpls/day.html"? 



I suppose that these url is bad...

So... your tmpl_path maybe is "bootstrap-calendar-master/tmpls/" or maybe "tmpls/"... or try with "bower_components/bootstrap-calendar/tmpls/" that seems correct

**Comment by finkikun** [comment]
Still nothing. 
In console I have got:
"Processing by EventsController#index as HTML
  Rendered events/index.html.erb within layouts/application (0.6ms)
Completed 200 OK in 3ms (Views: 2.6ms | ActiveRecord: 0.0ms)"

**Comment by eleazan** [comment]
You have a public url? :/

**Comment by finkikun** [comment]
No, I don't. For now I try to run it on localhost.

**Comment by eleazan** [comment]
And... what framework are u using?

**Comment by finkikun** [comment]
Rails 4.2.4

**Comment by eleazan** [comment]
So, can you do that tests? 

Go to your localhost page with calendar...

1) Show me the local url (like: localhost/test/calendar/index.html)
2) Open the console, and type 
````
$("#calendar").length
````
And say what do you get (if it's 0, it than #calendar div does not exists!)
3) Execute that:
````
$("#calendar").calendar({
    events_source: [{
        "id": 293,
        "title": "Event 1",
        "url": "http://example.org",
        "class": "event-important",
        "start": 12039485678000,
        "end": 1234576967000
    }],
    tmpl_path: "bower_components/bootstra-calendar/tmpls"
});
````
And see if an error is showed

**Comment by finkikun** [comment]
1) http://localhost:8080/events
2) undefined
3) TypeError: $(...).calendar is not a function

**Comment by eleazan** [comment]
So....

2) #calendar (a div with that id!) no exists...
3) calendar.js is not loaded >_>

**Comment by finkikun** [comment]
Ok, so here goes n00b question:
If I have only scaffold Event and all this calendar stuff, where should I define #calendar?

**Comment by eleazan** [comment]
"all this calendar stuff"... but, seems to not loading "that" stuff... 

can you open that?
http://localhost:8080/bower_components/jquery/jquery.min.js ?
http://localhost:8080/bower_components/bootstra-calendar/calendar.js ?

**Comment by finkikun** [comment]
Yep, I got your point. Indeed, routes seem to not exist, but paths are correct (I mean, they refer to actual paths which are for some reason incorrect). This error occur in console when I change 'href' to 'src' - when it's 'href', there's no error.

Is it because I did not move them from bower_components to another folder?

**Comment by eleazan** [comment]
No idea...

Seems to be a Rails thing... any "public folder", or how it manage the resources... how do you add a normal css?
